docs/hive.md

```diff
@@ -35,3 +35,34 @@ Installing then simply requires the running of the install script.
 ```bash
 ./hack/hive-dev-install.sh
 ```
+
+
+## Testing own hive build
+
+1. Create a fork of `github.com/openshift/hive` on github
+    1. Make your changes and push into your fork.
+1. Build and push a custom Hive image:
+    ```bash
+    # From a hive repo checkout with your changes
+    export IMG="quay.io/{username}/hive:latest"
+    GOOS=linux GOARCH=amd64 make build image-hive-fedora-dev docker-push
+    ```
+1. Make sure that your image is public so AKS can pull it
+1. Update `./hack/hive-generate-config.sh`:
```

Comment:
Since this file is committed to source control it may make more sense to pull these values from the local environment rather than changing the script itself, which may lead to accidental updates. See comment on script.